hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-cs.tex

```diff
@@ -1,33 +1,27 @@
-% This file has been converted for the hyph-utf8 project from czhyphen.tex and
-% czhyphen.ex, whose author has been identified as Pavel Ševeček <pavel at
-% lingea.cz>.  The licence terms are unchanged.
+% title: Czech hyphenation patterns
+% copyright: Copyright (c) Pavel Ševeček
+%     Lingea s.r.o., Brno, Czech Republic
+%     E-mail pavel@lingea.cz
```

Comment:
Emails are sometimes slightly obfuscated, sometimes not, in this case email is a separate line, in other cases it is not. I prefer obfuscation to give the authors at least a tiny bit of protection.

Reply:
Adopted.